src/app/app.module.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import uiRouter from '@uirouter/angularjs';
+import helloComponent from './hello/hello';
+import worldComponent from './world/world';
+
+const app = angular.module('app', [uiRouter]);
+
+app.component('hello', helloComponent);
+app.component('world', worldComponent);
+
+app.config(['$stateProvider', function (provider) {
```

Comment:
Just fyi, this routing/states configuration is on its own file and imported on module.js
maybe for the future is a good refactoring 👍